### DISCUSSION TITLE
N/A

### DISCUSSION THREAD

**Discussion by ghost** [has2k1/plotnine#846] [2024 Plotnine Contest]

**Comment by NatashaData** [comment]
Hi Ryan, nice work & interesting data set!
I think, considering your legend, you could take the extra text out of the graph and just leave the country name. Even better if you had arrows to the ones close together so it's more legible.

**Reply by ghost** [reply]
Thank you for the feedback! I took your suggestion and made some updates!

**Comment by has2k1** [comment]
@ghost, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you for your attempt.
